### PR TITLE
Added Link to pofiles.md in readme.md  

### LINKED ISSUE #1052: Add link to `README.md` that links to the `profiles.md` file
In order for everyone to see the awesome github profiles, a link to the `profiles.md` would be helpful. This is because I was confused as I was trying to find all the actual screenshots of everone's profiles, but I couldn't see them. I was going to put an issue up to ask where they all were, but then searched through the file directories, and found the `profiles.md` file with all the screenshots.
If this is possible, this is a beginner contributor issue, that anyone can contribute to.
Thank you.

### PR DESCRIPTION
**Pull request by hasnainmakada-99**
resolves #1052 
---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

## This PR adds/fixes
- The issue number #1052 in which I've added the link to profiles.md inside the main readme.md file
- Now the use will be able to see the profiles from the GitHub repository itself

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ Feel free to submit a [pull request](https://github.com/EddieHubCommunity/awesom
 
 4. Create a [Pull Request](https://github.com/EddieHubCommunity/awesome-github-profiles/pulls) to add the changes to this repository. Please change the title to `fix: add <your-username>` and in the description add `resolve #issueNumber`
 
+## Check out our awesome profiles [Here](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/profiles.md)
```
Comment (schmelto) [inline README.md:48]:
```suggestion
## Check out our awesome profiles [here](https://eddiehubcommunity.github.io/awesome-github-profiles/)
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Comment (hasnainmakada-99) [reply README.md:48]: Hey @schmelto can you please tell me what I need to fix

Comment (hasnainmakada-99) [reply README.md:48]: ok I've got It now

Comment (hasnainmakada-99) [reply README.md]: Hey @schmelto the typo error has been fixed now

**Comment by hasnainmakada-99** [comment]
Hey @schmelto can you please review my pr

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to me!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Panquesito7 — APPROVED** [review]

**Review by vikasganiga05 — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**